src/infinite-list/index.html

```diff
@@ -1,17 +1,17 @@
 <ul ref="wrapper" class="gb-list__wrapper {isGrid ? 'gb-list__wrapper--grid' : 'gb-list__wrapper--column'}">
-  <li data-is="gb-list-item" class="load first-loader" if="{$infinite.isFetchingBackward}" _consumes="infinite">
-    <gb-infinite-loader label="{$infinite.loaderLabel}"></gb-infinite-loader>
+  <li data-is="gb-list-item" class="load first-loader" label="{props.loaderLabel}" if="{props.isFetchingBackward}">
```

Comment:
Usually we don't pass down props to components that don't have those prop types in their definition (ie, `gb-list-item` doesn't expect a `label` prop). It technically is fine and works, but usually in these cases where a component needs context from higher up but doesn't use props related to that context in it's definition, we will use a `_consumes` to access that context. So please use `_consumes="infiniteList"` here and `$infiniteList.loaderLabel` in the infinite-loader label prop. I hope that makes sense